tuplex/io/CMakeLists.txt

```diff
@@ -13,21 +13,56 @@ endif ()
 find_package(LibMagic REQUIRED)
 message(STATUS "Found LibMagic ${LibMagic_INCLUDE_DIR}, ${LibMagic_LIBRARIES}")
 
-
 include_directories("include")
 include_directories(${Boost_INCLUDE_DIR})
 
+# Install and build ORC C++ APIs
+find_package(Protobuf REQUIRED)
+get_filename_component(Protobuf_HOME "${Protobuf_INCLUDE_DIRS}" DIRECTORY)
+
+include(ExternalProject)
+set(EXTERNAL_INSTALL_LOCATION ${CMAKE_BINARY_DIR}/third_party)
```

Comment:
maybe let's add a comment: ORC basically comes standardwise shipped with its own snappy, zip libs? Do we know anything about versions, or whether we could exchange them? Do we need to add any libs to the pip package so it works on some other machine? 

I feel a test of a pip package with orc support on a clean machine would be a good idea.

Reply:
has this been addressed?